_posts/rosi_bajari/2018-07-13-genomics.md

```diff
@@ -0,0 +1,102 @@
+
+
+There’s a few different aberrations that can appear in DNA that we regularly search for. These are some of the most popular types of DNA differences:
+
+-   SNPs (Single Nucleotide Polymorphisms), pronounced “snips”. This is where a *single* tiny base is changed in the DNA, which [sometimes has no effect and sometimes has a huge effect ](https://www.thisamericanlife.org/577/something-only-i-can-see/act-one).
+    
+-   Indels: sequences of 1-1000 bases that have been inserted or deleted from DNA
+    
+-   Copy Number Alteration: Sometimes a gene might get deleted, or copy-pasted into the genome multiple times. 
+    
+Once the differences are spotted, then the real fun begins! Bioinformaticians, biologists, computational biologists, clinicians and physicians are all interested in this data as these differences in DNA can often be the reason for a disease, can simply be a risk marker, or other times don't do anything at all.
+
+
+## You mentioned data portals - where do data portals come into this?
+
+The area of genomics has become so advanced that it's actually outpacing the very scientists who invented it! Since the cost of sequencing a genome has fallen so low, more and more projects are looking to sequencing, and there are even commercial companies that offer personal sequencing and clinical recommendations. Thats a lot of data! Where does all this data live, how do we protect it, and how do people look for it and access it? 
+
+That’s where data portals come in! Data portals help build and manage the infrastructure to:
+
+-   Store genomic data files    
+-   Let people search through it (without actually touching the data)   
+-   Download data  
+-   Work with genomic data in cloud computing environments
+-   Interact with data analytics in real-time analysis
+    
+We make it easy for scientists to find exactly what they need, and take care of hosting the data and its required infrastructure, saving them time and money so that they can focus on figuring out what the data is _actually_ telling us.
+
+On top of that, data portals ingest and display some of the mutation data so that researchers can look for very specific cases. Its alot easier to browse these mutations on a webpage than downloading files and blindly searching through them one-by-one hoping to find the mutation of interest.
+
+
+## Ok, so now you get what genomics is all about, but what’s so cool genomics?
+
+There’s a couple really ideas that excite me about genomics. First, personalized medicine is a *really* hot topic. Research has shown that many different things affect how we get sick or react to treatments, from our actual DNA, to the environmental modifications on our DNA, or even the bacteria we have in our guts. Since each person has this unique biology, the goal of precision (or personalized) medicine enabled by genomics is to get to the point where doctors can offer treatment profiles based on your specific sgenetic profile. Cool, right?
```

Comment:
`based on your specific sgenetic profile` -> `based on your specific genetic profile`

Reply:
1. You might want to say "I get" (switching it to the reader's perspective), or keep it as "you get" (keeping in the narrator's voice)... 
2. "what's so cool genomics" needs an "about" in there.

Reply:
"There’s a couple *of* really *interesting?powerful?important* ideas"